conftest.py

```diff
@@ -367,7 +447,87 @@ def fixture_dtest_setup(request,
         except Exception as e:
             logger.error("Error saving log:", str(e))
         finally:
+            if failed or reuse_option == Reuse_cluster.REUSE_CLUSTER_NO:
+                safe_cluster_cleanup(request, dtest_setup)
+                reusable_dtest_setup = None
+
+
+def cant_reuse_cluster_reason(request, dtest_setup, last_test_class, dtest_config):
+    if dtest_setup is None:
+        return "Can't reuse bc reusable cluster is None"
+
+    # Can't reuse if some nodes down
+    if dtest_setup is not None \
```

Comment:
Recommend rewriting this to something a bit more clear. For example:
```
    # Can't reuse if some nodes down
    live_node_count = len([node for node in dtest_setup.cluster.nodelist() if node.is_live()])
    if dtest_setup is not None and live_node_count != len(dtest_setup.cluster.nodelist()):
        return "Can't reuse cluster because number of live node count doesn't match"```